openedx_lti_tool_plugin/urls.py

```diff
@@ -1,10 +1,27 @@
 """URL configuration for openedx_lti_tool_plugin."""
-from django.urls import path
+from django.conf import settings
+from django.urls import include, path, re_path
 
 from openedx_lti_tool_plugin import views
 
 urlpatterns = [
-    path('1.3/login/', views.LtiToolLoginView.as_view(), name='lti1p3-login'),
-    path('1.3/launch/', views.LtiToolLaunchView.as_view(), name='lti1p3-launch'),
-    path('1.3/pub/jwks/', views.LtiToolJwksView.as_view(), name='lti1p3-pub-jwks'),
+    path('1.3/', include([
+        path('login', views.LtiToolLoginView.as_view(), name='lti1p3-login'),
+        path('pub/jwks', views.LtiToolJwksView.as_view(), name='lti1p3-pub-jwks'),
+        re_path(
+            fr'^launch/{settings.COURSE_ID_PATTERN}$',
```

Comment:
Can't this be imported from the lms?

Reply:
@Squirrel18 I will already use the setting from the LMS while installed, on the cli it will use the test settings, I could patch it on tests and not add it to the settings, so we don't need to keep a copy for tests.